### DISCUSSION TITLE
Allow to switch between tabs using CTRL+TAB and CTRL+SHIFT+TAB

### DISCUSSION THREAD

**Discussion by MarvinKlein1508** [dbeaver/dbeaver#21646] [General]
In almost every windows program you can switch between tabs using the shortcuts
CTRL+TAB and CTRL+SHIFT+TAB. This does not seem to be the case for DBeaver.

**Comment by ShadelessFox** [comment]
Hello @MarvinKlein1508,

You can switch between editor tabs using **Ctrl+PageUp** and **Ctrl+PageDown**.

You can change keys here:

![image](https://github.com/dbeaver/dbeaver/assets/35821147/f530be39-253c-402c-824f-40557742ad2c)

Please let me know if it solves your issue.

**Comment by MarvinKlein1508** [comment]
@ShadelessFox what about the inner tabs? 
![grafik](https://github.com/dbeaver/dbeaver/assets/32510006/1e6d7cec-af88-4122-a9ca-5704a0e66a55)

**Comment by ShadelessFox** [comment]
Same shortcuts, but with **Alt** instead of **Ctrl**: **Alt+PageUp** and **Alt+PageDown**.

Can be changed, too:

![image](https://github.com/dbeaver/dbeaver/assets/35821147/c9549521-6d7b-418f-8e5e-22ebc65cf8d7)

**Comment by MarvinKlein1508** [comment]
Thanks a lot! This makes much so much more productive!
